libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/history/ReplayHistoryMapper.kt

```diff
@@ -0,0 +1,46 @@
+package com.mapbox.navigation.core.replay.history
+
+import android.util.Log
+import com.google.gson.Gson
+import com.google.gson.internal.LinkedTreeMap
+
+typealias CustomEventMapper = (String, LinkedTreeMap<*, *>) -> ReplayEventBase?
+
+class ReplayHistoryMapper(
```

Comment:
@Guardiola31337 here's the new mapper that's a little customizable or replaceable